async/amqp/integration_test.go

```diff
@@ -0,0 +1,187 @@
+package amqp
```

Comment:
Should we add build tag (`// !build integration`) in this file also?

Reply:
🤦‍♂  sorry, I was testing something locally; re-added the correct build tag.